src/Symfony/Component/String/ByteString.php

```diff
@@ -25,20 +25,58 @@
  */
 class ByteString extends AbstractString
 {
+    private const ALPHABET_ALPHANUMERIC = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
```

Comment:
there are only 62 chars here, the previous had 64.
can you bench the previous implementation with the new one in the base64 case please?
we might decide to keep a fast path for base64, if it's faster.

Reply:
the previous had 64 but `+` and `/` were always being replaced by `''`, so it is way always 62

Reply:
right I forgot about this :)
what about the perf?

Reply:
not sure tbh, but I would assume the previous implmenetation have a better performance when it comes to using the alphanumeric ( no custome characters )